### PR TITLE
Feature: Improved design of the security property pages

### PR DESCRIPTION
**Pull request by onein528**
### Description

To adapt with new style, update Security and SecurityAdvanced pages. This changes is not one of security advanced page integration plan.

### Details of changes

**SecurityPage**
- Change full name displaying style from (FontWeight: `SemiLight`, FG: `Primary`) to (FontWeight: `Normal`, FG: `Secondary`) in an ACL.
- Larger add/remove ACL buttons (`Padding="8,8"`)
- `MenuFlyoutSeparator` to `Border` and color is set as `DividerStrokeColorDefaultBrush`
- Display 'Advanced permissions' on `>` button
- Change 'Change owner' `Button` to `HyperlinkButton`
- Align 'Change owner' left side

**SecurityAdvancedPage**
- Change full name displaying style from (FontWeight: `SemiLight`, FG: `Primary`) to (FontWeight: `Normal`, FG: `Secondary`) in an ACL.
- Larger add/remove ACL buttons (`Padding="8,8"`)
- `MenuFlyoutSeparator` to `Border` and color is set as `DividerStrokeColorDefaultBrush`
- Moveable `ListViewItem` columns implemented with `GridSplitter`
- Transparent `DropDownButton`
- Change the way of displaying type from Glyph to Text (`Allow` or `Deny`)

### Self-check list

- [ ] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
- [x] Did you build the app and test your changes?
- [x] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### Screenshots

Security page (*Old*, *New*)
image

Old SecurityAdvancedPage
image

New SecurityAdvancedPage
image

### CODE REVIEW

File: src/Files.App/Filesystem/Security/AccessControlEntry.cs [hunk 1]
```diff
@@ -30,10 +30,17 @@ public AccessControlType AccessControlType
...
+			{
+				AccessControlType.Allow => "Allow",
+				_ => "Deny" // AccessControlType.Deny
+			};
+
 		public string AccessControlTypeGlyph
 			=> AccessControlType switch
 			{
-				AccessControlType.Allow => "\xF13E",
+				AccessControlType.Allow => "\xE73E",
```
Comment (yaira2) [inline src/Files.App/Filesystem/Security/AccessControlEntry.cs:43]: Can you share the before and after?

Comment (onein528) [reply src/Files.App/Filesystem/Security/AccessControlEntry.cs:43]: Old|New
---|---
image|image

Previous one was smaller.

**Comment by yaira2** [comment]
Is this ready for review?

**Comment by onein528** [comment]
Yes

**Review by yaira2 — APPROVED** [review]
LGTM

File: src/Files.App/Strings/en-US/Resources.resw [hunk 2]
```diff
@@ -2664,6 +2664,8 @@
   <data name="ExitCompactOverlayDescription" xml:space="preserve">
     <value>Exit compact overlay</value>
   </data>
+  <data name="Change" xml:space="preserve">
+    <value>Change</value>
```
Comment (yaira2) [inline src/Files.App/Strings/en-US/Resources.resw:2668]: This is missing a closing tag

**Pull request merged**